frontend/src/pages/application/DeviceGroup/Settings/Environment.vue

```diff
@@ -19,8 +19,6 @@
                 <ExclamationCircleIcon class="ff-icon mr-2" />
                 <span class="relative top-0.5">Note: Updating environment variables can cause devices in the group to be restarted.</span>
             </span>
-
-            <ChevronRightIcon class="ff-icon align-self-right" />
```

Comment:
not related to the current scope, but was an easy fix to address as it was a leftover (had no reason to be render)

Reply:
rendered*